src/uhs/atomizer/atomizer/state_machine.cpp

```diff
@@ -99,6 +99,12 @@ namespace cbdc::atomizer {
             resp.value());
     }
 
+    void state_machine::commit_config(
+        const nuraft::ulong log_idx,
+        nuraft::ptr<nuraft::cluster_config>& /*new_conf*/) {
+        m_last_committed_idx = log_idx;
```

Comment:
Is it worth asserting here that `log_idx > m_last_committed_idx`, or even stronger that `log_idx == m_last_committed_idx + 1`?

Reply:
I agree about `log_idx == m_last_committed_idx + 1`. Will add this to both `commit()` and `commit_config()`